app/views/docs/functions.phtml

```diff
@@ -646,9 +646,13 @@ class MainActivity : AppCompatActivity() {
 
 <p>Appwrite allows your project's end-users to execute Cloud Functions using client API or your client SDK. Execution is permitted to any user who has been granted the "execute" permission in your Cloud Functions settings page. The execution permission can accept any of the typical Appwrite <a href="/docs/permissions">permission types</a>.</p>
 
-<p>When triggering a Cloud Function execution from the client, your users will be limited to a specific amount of execution per minute to make sure your Appwrite server is not being abused. The default limit is 60 calls per 1 minute. You can edit this limit using the server <a href="/docs/environment-variables#functions">environment variables</a>.</p>
+<p>To prevent abuse, each user is rate limited to making 60 calls per minute to the Functions API. Each execution has a default timeout of 15 seconds to prevent hanging functions from blocking resources. This timeout can be configured per function in a function's settings or in <span class="tag">appwrite.json</span> for up to 900 seconds.</p>
```

Comment:
We must also mention that rate limits are not applicable to server SDKs authenticated with an API key allowing for usecases that need to by pass the default rate limit

Reply:
Yes! This is important, great catch! I didn't know this :P